listenbrainz/db/dump_manager.py

```diff
@@ -280,6 +306,37 @@ def write_hashes(location):
             raise
 
 
+def sanity_check_dumps(location, expected_count):
+    """ Sanity check the generated dummps to ensure that none are empty
+
+    Args:
+        location (str): the path in which the dump archive files are present
+        expected_count (int): the number of files that are expected to be present
+    Return:
+        boolean: true if the dump passes the sanity check
+    """
+
+    count = 0
+    for file in os.listdir(location):
+        try:
+            dump_file = os.path.join(location, file)
+            print(dump_file)
```

Comment:
No use of logger here?

Reply:
The loggers are not available in this context, so I would have to create another logger for some of the code, some of the time. Not what I really want to be doing right now.